### ISSUE TITLE
jsx-wrap-multilines for array declaration

### ISSUE THREAD

**Issue by CzBuCHi** [yannickcr/eslint-plugin-react#1713]
not sure it it is correct, but i would like to configure lint to not show error for this code:
```ts
return [
  <div>
    ...
  </div>,
  <div>
    ...
  </div>,
];
```
currently it complains about `jsx-wrap-multilines` rule and "correct" code looks like this:

```ts
return [
  (
  <div>
    ...
  </div>
  ),
  (
  <div>
    ...
  </div>
  ),
];
```
but vscode autoformat will remove these redundant brackets 

anyway: what is "correct" style for this?

**Comment by ljharb** [comment]
```jsx
return [
  (
    <div>
      ...
    </div>
  ),
  (
    <div>
      ...
    </div>
  ),
];
```
^ is in fact the correct code for it; I would recommend disabling any autoformat that isn't eslint.
